src/_newsletters/2019-09-09-newsletter.md

```diff
@@ -0,0 +1,99 @@
+---
+layout: newsletter
+asset-type: newsletter
+name: newsletter-september-2019
+title: 'Newsletter - September 2019'
+date: 2019-09-09 16:00:00 +00:00
+author: Emma Booth
+image:
+  src: /assets/custom/img/newsletter/newsletter-default.png
+tags:
+- software craftsmanship
+- TDD
+- testing
+- newsletter
+---
+
+Hello,
+
+It has been a busy few months for the team and we're pleased to bring the newsletter back, including some useful resources for you to take advantage of, such as an article on 'The Fine Art of Fast Development', a talk on building 'Sustainable and Empathetic Cross-Functional Teams' and, if you haven't already checked it out yet, Katalyst(https://katalyst.codurance.com/), our new handy skill-honing tool. Please feel free to share the newsletter with your colleagues for their own learning and development! 
```

Comment:
There are few links like the Kayalyst one above that are in the newsletter. 
Is the URL supposed to be shown in the newsletter? 
Katalyst(https://katalyst.codurance.com/)

Or should they be hyperlinks from the text?
[Katalyst](https://katalyst.codurance.com/)

Reply:
Yes they should be :( How do I change it? 
Sorry to be a pain!

Reply:
https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#links I think you just have to wrap the text you want to be a hyperlink within [] like you've done with the titles. Once you've done that re-send me the file and I'll update it. It seems as though I can also edit it directly in the browser here https://github.com/codurance/site/edit/features/newsletter/src/_newsletters/2019-09-09-newsletter.md?pr=%2Fcodurance%2Fsite%2Fpull%2F1442 but I'm not sure if you have permissions

Reply:
RATS! I'll do that now, thank you for your patience. It would be good to spend some time (whenever you're free) showing me how to do these first steps myself for next time and therefore make sure I can pull my weight before asking for it to be checked :D